pyblish/cli.py

```diff
@@ -355,7 +355,7 @@ def gui(ctx, package):
 
     with _cli_plugin(data=context.data) as plugin_path:
         environ["PYBLISHPLUGINPATH"] = os.pathsep.join(
-            ctx.obj["plugin_paths"] + [plugin_path]
+            ctx.obj["plugin_paths"] + tuple(plugin_path)
```

Comment:
FYI these are not the same.

```py
>>> tuple("hello")
('h', 'e', 'l', 'l', 'o')
>>> ["hello"]
['hello']
>>>
```

Are we sure `plugin_path` is not a string?

Reply:
I believe this was causing us some major headaches at one point. @illicitit might remember exactly.

Reply:
if i remember well it was because `ctx.obj["plugin_paths"]` is tuple and concatenation of tuple and list is not allowed. But you got true it's not right, should be `tuple([plugin_path, ])`

Reply:
maybe just `(plugin_path, )`